data/json/items/ranged/pneumatic.json

```diff
@@ -15,9 +15,8 @@
     "flags": [ "NEVER_JAMS", "NON_FOULING" ],
     "skill": "rifle",
     "ammo": [ "BB" ],
-    "weight": "1000 g",
+    "weight": "1723 g",
     "volume": "1500 ml",
-    "longest_side": "90 cm",
```

Comment:
dont forget to remove the changes to "bbgun" in pneumatic

Reply:
What'd you mean by this?